css/templates/travel/page.css

```diff
@@ -30,15 +31,10 @@
 @import 'basscss-border';
 @import 'basscss-hide';
 
-/* BassCSS Add-ons */
+@import '../../ampstart-base/base-page-vars';
```

Comment:
you will not need all these when you include core - https://github.com/ampproject/ampstart/blob/master/css/core.css